doc/decisions/simplify_api.md

```diff
@@ -48,6 +48,82 @@ However, the API changes follow these rules:
 - _Remove_ all functions that can be replaced by others (e.g. `ksPop`)
 - _Rename_ all functions to start with `elektra` as described in [Elektra Prefix](elektra_prefix.md), and use `KeySet` instead of `ks` (also applies to names above)
 
+Afterwards the Elektra Core Public API would look as follows:
+
+#### KDB
+
+- kdbOpen
+- kdbClose
+- kdbGet
+- kdbSet
+
+#### Key
+
+- keyNew
+- keyVNew
+- keyCopy
+- keyClear
+- keyDel
+- keyIncRef
+- keyDecRef
+- keyGetRef
+- keyLock
+- keyIsLocked
+- keyDup
+
+#### Meta
+
+- keyMeta
+
+#### Tests
+
+- keyCmp
+- keyNeedSync
+- keyIsBelow
+- keyIsBelowOrSame
+- keyIsDirectlyBelow
+
+#### Name
+
+- keyName
+- keySetName
+- keyAddName
+- keyNameSize
+- keyUnescapedName
+- keyUnescapedNameSize
```

Comment:
```suggestion
- keyEscapedName
- keyEscapedNameSize
```

Since we move away from escaped name as the default, the unescaped name will be returned by `keyName` and so we need other functions for the escaped name.